### DISCUSSION TITLE
Accuracy rounding

### DISCUSSION THREAD

**Discussion by SevenChords** [ppy/osu#19869] [Ideas] [answered]
The accuracy rating is always rounded down it seems like, if you take the screenshot below as an example (mania with hold off mod active, meaning no sliders), calculating the accuracy manually (calculation below screenshot) you get a result of 95.53**9**%, which i think should round up to 95.54% accuracy on the final score, but instead of rounding to 2 decimal places it cuts off anything past 2 decimal places.

![Screenshot 2022-08-20 at 10 55 40](https://user-images.githubusercontent.com/31819497/185737343-5db1497d-e694-43f3-8e1e-7ddd2cb12b72.png)

Manual Accuracy Calculation:

```
( 315 x 375
+ 300 x 191
+ 200 x 32
+ 100 x 1
+ 50  x 3
+ 0   x 3   )
devide by
( 315 x 605 )
result
0.955398
```

**Comment by plaguemask** [comment, marked as answer]
It used to round, but it was changed because it caused problems.

If one were to score 89.995%, they'd correctly get a B grade. But everywhere the acc was shown would round it to 90.00%, which should correspond to an A grade. The most egregious example of this mismatch was how accuracies of 99.995% and higher incorrectly rounded to 100.00%. (See https://github.com/ppy/osu/pull/12181)

Switching from always rounding to always flooring was the easy way to avoid all that.
